circle.yml

```diff
@@ -1,3 +1,11 @@
+machine:
+  node:
+    version: 6.10.2
```

Comment:
are you 100% sure this needs to be moved from below to here?

Reply:
No, i haven't tested the it the other way😅